.eleventy.js

```diff
@@ -32,7 +32,15 @@ module.exports = (config) => {
   const nunjucksEnv = createNunjucksEnvironment([join(__dirname, 'src/site/_includes')])
   config.setLibrary('njk', nunjucksEnv)
   config.addNunjucksFilter('rejectattr_path', (array, propertyPath) => (
-    array.filter((item) => !get(item, propertyPath))
+    array && array.filter((item) => !get(item, propertyPath))
+  ))
+
+  config.addNunjucksFilter('selectattr_path', (array, propertyPath) => (
+    array && array.filter((item) => get(item, propertyPath))
+  ))
+
+  config.addNunjucksFilter('getUrlInCollection', (array, url) => (
+    array && array.find((item) => item.url === url)
```

Comment:
If we only need to check if it's in the array might be nicer to rename to `isUrlInCollection` and cast the result to a Boolean (and remove the `not not` from the template).

Reply:
Did you mean to reply to this – can see the comment is marked as resolved.